scripts/build_energy_totals.py

```diff
@@ -1153,32 +1153,31 @@ def build_transport_data(
     ----------
     - Swiss transport data: `BFS <https://www.bfs.admin.ch/bfs/en/home/statistics/mobility-transport/transport-infrastructure-vehicles/vehicles/road-vehicles-stock-level-motorisation.html>`_
     """
+    years = np.arange(2000, 2022)
 
     # first collect number of cars
     transport_data = pd.DataFrame(idees["passenger cars"])
 
     countries_without_ch = set(countries) - {"CH"}
     new_index = pd.MultiIndex.from_product(
-        [countries_without_ch, transport_data.index.levels[1]],
+        [countries_without_ch, transport_data.index.unique(1)],
         names=["country", "year"],
     )
 
     transport_data = transport_data.reindex(index=new_index)
 
     if "CH" in countries:
         fn = snakemake.input.swiss_transport
-        swiss_cars = pd.read_csv(fn, index_col=0).loc[2000:2023, ["passenger cars"]]
+        swiss_cars = pd.read_csv(fn, index_col=0).loc[years, ["passenger cars"]]
```

Comment:
if we update the eurostat data this would break with years. The swiss data is only available for the years 2000:2023

Reply:
Ah, but if we hard-code (which I want to avoid in the first place), we will run into the same error again as raised in #1218, so I would rather let the code fail where the discrepancy of the input data is occurring. what do you think?

Reply:
Alright, then let's keep it as it is.